scripts/addon/build.py

```diff
@@ -21,6 +21,105 @@
 }
 
 
+# TODO: THIS ABSOLUTELY SHOULD NOT BE HERE, SHOULD USE THE ONE IN SHARED, BUT IT IS NOT WORKING
```

Comment:
Not sure how frowned upon this would be…

```python
sys.path.insert(
    1, os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir, "utils"))
)
from generate_strings import parseYAMLTranslationStrings
```

Reply:
Worth noting that the script is missing an `import yaml` to work in its current form.

Reply:
This is now fixed.